torch/_dynamo/variables/misc.py

```diff
@@ -380,10 +385,16 @@ def __init__(self, inspected: VariableTracker, **kwargs) -> None:
 
     def var_getattr(self, tx: "InstructionTranslator", name: str) -> "VariableTracker":
         if name == "parameters":
+            if isinstance(self.inspected, UserMethodVariable):
```

Comment:
Should this go in the __init__ of `InspectSignatureVariable`? There is an if condition on `UserFunctionVariable`